client/lib/checkout/validation.js

```diff
@@ -85,10 +85,44 @@ export function tefPaymentFieldRules() {
 	);
 }
 
+/**
+ * Returns the token validation rule set
+ * @returns {object} the ruleset
+ */
+export function tokenFieldRules() {
+	return {
+		name: {
+			description: i18n.translate( 'Name on Card', {
+				context: 'Upgrades: Card holder name label on credit card form',
+			} ),
+			rules: [ 'required' ],
+		},
+
+		country: {
+			description: i18n.translate( 'Country' ),
+			rules: [ 'required' ],
+		},
+
+		'postal-code': {
+			description: i18n.translate( 'Postal Code', {
+				context: 'Upgrades: Postal code on credit card form',
+			} ),
+			rules: [ 'required' ],
+		},
+
+		'tokenized_payment_data': {
+			description: i18n.translate( 'Tokenized Payment Data', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?